task6/Working with posts.js

```diff
@@ -1,15 +1,15 @@
 (function(){
-class post {
-	constructor(id, avatar, author, createdAt, description, hashTags=[], photoLink="") {
+class Post {
+	constructor(id, avatar, author, createdAt, description, hashTags=[], photoLink="", likes = []) {
 		this.id = id;
 		this.avatar = avatar;
 		this.author = author;
 		this.createdAt = createdAt;
 		this.description = description;
 		this.photoLink = photoLink;
 		this.hashTags = hashTags;
-		this.likes=[];
-		this.countOfLikes=0;
+		this.likes=likes;
+		this.countOfLikes=this.likes.length;
```

Comment:
I meant, that field is redundant, since it can always be obtained from likes array